proxysql/tests/conftest.py

```diff
@@ -66,8 +66,15 @@ def dd_environment():
                 'TMP_DATA_DIR': tmp_dir,
             },
             conditions=[
-                CheckDockerLogs('db', ["mysqld: ready for connections"], wait=5),
-                CheckDockerLogs('proxysql', ["read_only_action RO=0 phase 3"], wait=5),
+                CheckDockerLogs(
+                    'db',
+                    [
+                        "MySQL init process done. Ready for start up.",
+                        "mysqld: ready for connections",
+                        "read_only_action RO=0 phase 3",
+                    ],
+                    wait=5,
+                ),
```

Comment:
I'm not sure why we moved the `read_only_action RO=0 phase 3`. Is this log generated by the mysql container and not the proxysql one?

Reply:
Good catch, I don't know how this was passing as it is. I guess it's generated in both places? :/

Reply:
Maybe, I don't know anything about proxysql tbh